include/seqan3/alignment/all.hpp

```diff
@@ -0,0 +1,61 @@
+//                 SeqAn - The Library for Sequence Analysis
+// ============================================================================
+//
+// Copyright (c) 2006-2018, Knut Reinert & Freie Universitaet Berlin
+// Copyright (c) 2016-2018, Knut Reinert & MPI Molekulare Genetik
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//
+//     * Redistributions of source code must retain the above copyright
+//       notice, this list of conditions and the following disclaimer.
+//     * Redistributions in binary form must reproduce the above copyright
+//       notice, this list of conditions and the following disclaimer in the
+//       documentation and/or other materials provided with the distribution.
+//     * Neither the name of Knut Reinert or the FU Berlin nor the names of
+//       its contributors may be used to endorse or promote products derived
+//       from this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL KNUT REINERT OR THE FU BERLIN BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+// LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+// OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+// DAMAGE.
+//
+// ============================================================================
+
+/*!\file
+ * \brief Meta-header for the \link align alignment module \endlink.
+ * \author Rene Rahn <rene.rahn AT fu-berlin.de>
+ */
+
+#pragma once
+
+/*!\defgroup align Alignment
+ * \brief Contains alignment functionality.
+ *
+ * The align module contains concepts, functions and classes that are related to computation of
+ * pairwise and multiple sequence alignments.
+ *
+ * \todo Write detailed landing page.
+ */
+
+/*!\namespace seqan3::align
+ * \brief The SeqAn3 namespace for alignment related functions and classes.
+ *
+ * SeqAn3 implements various algorithms that are specific to bioinformatics and uses special configurations
+ * for these. In order to avoid collisions of configurations or classes and functions all alignment specific
+ * functions, concepts, configurations etc. are contained in their own namespace.
+ *
+ * \attention
+ * This means you cannot use them, unless you explicitly add `using namespace seqan3::align;` (in addition
+ * to `using namespace seqan3;`).
```

Comment:
I added this paragraph to the `seqan3::literal` namespace because there actually is no way to use the literals without the using decleration, but I don't think we need to add this for every sub-namespace. It should be clear that it's members can be used without an using decleration, simply by prepending `align::`.